bootstrap.yml

```diff
@@ -1,5 +1,13 @@
 ---
 
+- hosts: local
+  gather_facts: False
+  tasks:
+    - name: Ask confirmation before running bootstrap playbook to prevent deleting project accidentally 
+      pause:
+        prompt: "This playbook will destruct existing project. Press enter to continue, Ctrl+C to interrupt"
```

Comment:
would be nice to display the name of the project that will be destroyed and the url of the platform ?... You might be mistaken on the environment variables you passed...

Reply:
what do you mean by `url of the platform` ? The OpenShift admin console ?

Reply:
If I add the information you ask, and I think this information are a good thing, I need first to compute all variables used in the project by running the task `set_vars.yml`. Doing this many thing will be prompt to user before asking him to continue or not.
Should I do this or not ?

Reply:
1)I meant the domain of the OpenShift platform so you know dev from prod?
2) I would say: if you don't display this info, and with the sentence you wrote, I understand it as a generic warning but not necessarily that you HAVE FOUND an existing project with this name.

I would start the warning by: "There is already a project with the name "foo-bar"..."

Reply:
We don't know if the project exists at this step. And we will don't know if it exists we just ensure that the project is absent, we don't delete it only if present.
So yes I can compute the good value after executing the `set_vars` task and then display the project and server we will work on.